docs/rst/getting_started/getting_started.rst

```diff
@@ -58,8 +54,9 @@ This allows to create a non limited and highly scalable decentralized and distri
 .. figure:: /rst/figures/ddsrouter_overview_wan.png
 
 
+#################
 Usage Description
-=================
+#################
```

Comment:
I would use the older section level, or separate them in different pages

Reply:
:+1: